core-ui/src/shared/components/K8sResourceSelect.js

```diff
@@ -88,7 +85,7 @@ export function K8sResourceSelect({
     } else return undefined;
   };
 
-  return allowTyping ? (
+  return (
     <div className="combobox--full-width">
       <ComboboxInput
```

Comment:
Please use the combobox with the [showAllEntries](https://github.com/SAP/fundamental-react/blob/main/src/ComboboxInput/ComboboxInput.js#L654) and [searchFullString](https://github.com/SAP/fundamental-react/blob/main/src/ComboboxInput/ComboboxInput.js#L643) flags enabled. (The documentation is not yet on the official site, since we don't have an official release yet, so please check the comments in the code, linked above)

I'd also suggest using the `selectionType='auto-inline'`, but that's up to you to decide.

Reply:
With `showAllEntries` flag it works not so well

![image](https://user-images.githubusercontent.com/46759954/136017774-bf4c372a-5921-44ef-aa8d-862ae95cdb07.png)

without this flag searching matching entries works better

![image](https://user-images.githubusercontent.com/46759954/136018091-f3293eb9-02c6-411e-af42-032adc143c68.png)

Reply:
We want to see all entries and matching entries on top or only matching entries?

Reply:
we should see all entries with the matching ones on top. the matching string should also be **bold**, I don't know why it's not, we could create a bug for that.